packages/noya-state/src/layers.ts

```diff
@@ -68,6 +68,12 @@ export const isSymbolInstance = (
   return layer._class === 'symbolInstance';
 };
 
+export const isInferredLayout = (
```

Comment:
I would move this into a new file `groupLayout.ts` in the same directory. Everything in this file takes `Layer` types as the first parameter. We'll probably have other layout-related functions in the future